apps/marketplace/components/Opportunity/Opportunity.js

```diff
@@ -251,7 +260,7 @@ const Opportunity = props => {
                 <div className="col-xs-12 col-sm-4">
                   <strong>Budget range</strong>
                 </div>
-                <div className="col-xs-12 col-sm-8">{brief.budgetRange || 'None specified'}</div>
+                <div className="col-xs-12 col-sm-8">{brief.budgetRange && brief.budgetRange}</div>
```

Comment:
This will be an empty div if there is no `budgetRange`, but the "Budget range" row itself will still show. Should we use a `{brief.budgetRange && (...` which wraps around this entire row? or add the check to the existing `brief.lotSlug === 'rfx'` above.

Reply:
I've added the check up higher so it hides the row (like it does for specialist opportunities).